src/vma/lwip/tcp.h

```diff
@@ -483,6 +488,11 @@ err_t lwip_tcp_event(void *arg, struct tcp_pcb *pcb,
 /*Initialization of tcp_pcb structure*/
 void tcp_pcb_init (struct tcp_pcb* pcb, u8_t prio);
 
+#ifdef DEFINED_EXTRA_STATS
+/* Set pointer to extra TCP stats instance */
+void register_tcp_stats_instance(struct tcp_pcb *pcb, socket_tcp_stats_t *stats);
+#endif /* DEFINED_EXTRA_STATS */
```

Comment:
`register_` notation is for global function mostly related vma_lwip() initialization.
I would suggest to name as `tcp_stat()`